dependencies/istio/install/README.md

```diff
@@ -1,6 +1,6 @@
 # Istio manifest for Kubeflow
 
-The manifest is taken from OSS Istio release 1.0.2.
+The manifest is taken from OSS Istio pre-release 1.1-20190111-09-15.
```

Comment:
Can you provide a link where these manifests are obtained?

Reply:
done